charts/policy-reporter/Chart.yaml

```
@@ -6,13 +6,13 @@ description: |
 
 type: application
 version: 1.7.3
-appVersion: 1.7.0
+appVersion: 1.8.0
```

You changed the appVersion instead of the Chart Version
```suggestion
appVersion: 1.7.3
```